### DISCUSSION TITLE
Where is my application by email？

### DISCUSSION THREAD

**Discussion by Melendez1209** [community/community#19692] [GitHub Education]
My current situation is like this. I applied for Github Education on June 25th. I should have received the result of my application by email yesterday (June 28). But to this day I'm still in a "processing" state.

**Comment by melvincwng** [comment]
@Melendez1209, for this issue, kindly raise a ticket to [GitHub Education Support](https://support.github.com/contact/education) over here for them to investigate.

There have been other users who reported the same issue as you. So not to worry, you are not the only one facing this.

Hope this clarifies.

**Reply by melvincwng** [reply]
@Melendez1209 Hey Mark, if this answer is helpful, please kindly 'Mark as answer' and follow me on GitHub :)

Appreciate the support, and have a great day ahead 😊
